ezgui/src/canvas.rs

```diff
@@ -161,10 +163,18 @@ impl Canvas {
                 let displacement_unit_x = displacement_x / displacement_magnitude;
                 let displacement_unit_y = displacement_y / displacement_magnitude;
                 // Add displacement along each axis
-                self.cam_x += displacement_unit_x * PAN_SPEED;
```

Comment:
You can just update cam_x and cam_y here like before. `UpdateType::Pan` doesn't need to return dx and dy at all, and `move_camera` can go away.

What's happening now is that you return the dx and dy, then call `move_camera`.

Reply:
Isn't a big part of what we're doing here moving the timing of the panning onto `runner.rs` vs doing it in `canvas.rs`?

Reply:
Sort of? I think of `runner` as controlling the rate that events happen, and `canvas` as using those events to do the panning. I'm in favor of doing the work inside `canvas` because it's less jumping back and forth to figure out what's happening.